spec/support/api_spec_helper.rb

```diff
@@ -16,19 +16,12 @@ module ApiSpecHelper
     "Accept"       => "application/json"
   }
 
-  API_STATUS = Rack::Utils::HTTP_STATUS_CODES.merge(0 => "Network Connection Error")
-
-  def parse_response
-    @code    = response.status
-    @result  = case @code
-               when Rack::Utils.status_code(:no_content)   then {}
-               when Rack::Utils.status_code(:unauthorized) then response.body
-               else JSON.parse(response.body)
-               end
-    @success = response.success?
-    @status  = API_STATUS[@code] || (@success ? Rack::Utils.status_code(:ok) : Rack::Utils.status_code(:bad_request))
-    @message = @result.kind_of?(Hash) ? @result.fetch_path("error", "message").to_s : @result
-    @success
+  def result
```

Comment:
I like `response_hash`...similar to `response_body`, but clearly converted to a Hash, however it might not match the `unauthorized` case.